app/src/main/java/com/mobileproto/hireddit/hireddit/WordToSpeech.java

```diff
@@ -0,0 +1,37 @@
+package com.mobileproto.hireddit.hireddit;
+
+import android.content.Context;
+import android.speech.tts.TextToSpeech;
+
+import java.util.Locale;
+
+/**
+ * TextToSpeech includes a method which will convert text to corresponding audio output
+ * TextToSpeech constructor gets the application context
+ * Speak() takes a String to generate an audio output
+ * TextToSpeech is able to speak out different languages, but for this project, we set it to English
+ */
+public class WordToSpeech {
+    private TextToSpeech m_ttobj;
```

Comment:
NORA LOVES COMMENTING ON LUCY'S ACCOUNT: why is this named such?
LUCY: I agree, also no underscores